pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+#Creating eve cgroup which will be parent cgroup for all eve services
+CGROUPS="cpuset cpu cpuacct blkio memory devices freezer net_cls perf_event net_prio hugetlb pids systemd "
+for cg in $CGROUPS; do
+  mkdir -p /sys/fs/cgroup/${cg}/eve
+done
+
+#Converts K, M, G values to Bytes
+function get_bytes {
+        if [ "${1: -1}" == "K" ]; then
+                echo $((${1%?} * 1024 ))
+        elif [ "${1: -1}" == "M" ]; then
+                echo $((${1%?} * 1024*1024 ))
+        elif [ "${1: -1}" == "G" ]; then
+                echo $((${1%?} * 1024*1024*1024 ))
+        else
+                echo $1
+        fi
+}
+
+eve_cgroup_memory_limit=$(get_bytes $(cat /proc/cmdline | grep -o '\bdom0_mem=[^, ]*' | cut -d = -f 2))
+eve_cgroup_memory_soft_limit=$(get_bytes $(cat /proc/cmdline | grep -o "\bdom0_mem=[^,]*,max:[^ ]*" | cut -d : -f 2))
+eve_cgroup_cpus_limit="0-$(cat /proc/cmdline | grep -o '\bdom0_max_vcpus=[^ ]*' | cut -d = -f 2)"
+
+if [ -z "${eve_cgroup_memory_limit}" ]; then
+        echo "Setting default value of 1000000000 for eve_cgroup_memory_limit"
```

Comment:
Please turn these constants into VARIABLES and declare them at the top of the script @adarsh-zededa

Reply:
Added a variable.